Compile errors with libupnp 1.14.13

Pull the latest changes and came across this when trying to rebuild.

```In file included from webserver.c:44:
webserver.c: In function ‘webserver_get_info’:
upnp_compat.h:32:131: error: parameter name omitted
   32 | #define VD_GET_INFO_CALLBACK(NAME, FILENAME, INFO, COOKIE) int NAME(const char* FILENAME, UpnpFileInfo* INFO, const void* COOKIE, const void **)
      |                                                                                                                                   ^~~~~~~~~~~~~
webserver.c:146:8: note: in expansion of macro ‘VD_GET_INFO_CALLBACK’
  146 | static VD_GET_INFO_CALLBACK(webserver_get_info, filename, info, cookie)
      |        ^~~~~~~~~~~~~~~~~~~~
webserver.c: In function ‘webserver_open’:
upnp_compat.h:33:149: error: parameter name omitted
   33 | ne VD_OPEN_CALLBACK(NAME, FILENAME, MODE, COOKIE) UpnpWebFileHandle NAME(const char* FILENAME, enum UpnpOpenFileMode MODE, const void* COOKIE, const void
)
      |                                                                                                                                                ^~~~~~~~~~~

webserver.c:172:8: note: in expansion of macro ‘VD_OPEN_CALLBACK’
  172 | static VD_OPEN_CALLBACK(webserver_open, filename, mode, cookie)
      |        ^~~~~~~~~~~~~~~~
webserver.c: In function ‘webserver_read’:
upnp_compat.h:34:148: error: parameter name omitted
   34 | ine VD_READ_CALLBACK(NAME, HANDLE, BUFFER, LENGTH, COOKIE) int NAME(UpnpWebFileHandle HANDLE, char* BUFFER, size_t LENGTH, const void* COOKIE, const void
)
      |                                                                                                                                                ^~~~~~~~~~~

webserver.c:199:8: note: in expansion of macro ‘VD_READ_CALLBACK’
  199 | static VD_READ_CALLBACK(webserver_read, fh, buf, buflen, cookie)
      |        ^~~~~~~~~~~~~~~~
webserver.c: In function ‘webserver_write’:
upnp_compat.h:34:148: error: parameter name omitted
   34 | ine VD_READ_CALLBACK(NAME, HANDLE, BUFFER, LENGTH, COOKIE) int NAME(UpnpWebFileHandle HANDLE, char* BUFFER, size_t LENGTH, const void* COOKIE, const void
)
      |                                                                                                                                                ^~~~~~~~~~~

upnp_compat.h:35:32: note: in expansion of macro ‘VD_READ_CALLBACK’
   35 | #define VD_WRITE_CALLBACK(...) VD_READ_CALLBACK(__VA_ARGS__)
      |                                ^~~~~~~~~~~~~~~~
webserver.c:218:8: note: in expansion of macro ‘VD_WRITE_CALLBACK’
  218 | static VD_WRITE_CALLBACK(webserver_write, fh, buf, buflen, cookie)
      |        ^~~~~~~~~~~~~~~~~
webserver.c: In function ‘webserver_seek’:
upnp_compat.h:36:145: error: parameter name omitted
   36 | define VD_SEEK_CALLBACK(NAME, HANDLE, OFFSET, ORIGIN, COOKIE) int NAME(UpnpWebFileHandle HANDLE, off_t OFFSET, int ORIGIN, const void* COOKIE, const void
)
      |                                                                                                                                                ^~~~~~~~~~~

webserver.c:223:8: note: in expansion of macro ‘VD_SEEK_CALLBACK’
  223 | static VD_SEEK_CALLBACK(webserver_seek, fh, offset, origin, cookie)
      |        ^~~~~~~~~~~~~~~~
webserver.c: In function ‘webserver_close’:
upnp_compat.h:37:104: error: parameter name omitted
   37 | #define VD_CLOSE_CALLBACK(NAME, HANDLE, COOKIE) int NAME(UpnpWebFileHandle HANDLE, const void* COOKIE, const void *)
      |                                                                                                        ^~~~~~~~~~~~
webserver.c:250:8: note: in expansion of macro ‘VD_CLOSE_CALLBACK’
  250 | static VD_CLOSE_CALLBACK(webserver_close, fh, cookie)
      |        ^~~~~~~~~~~~~~~~~
make[3]: *** [Makefile:429: webserver.o] Error 1
make[3]: Leaving directory '/home/pi/gmrender-resurrect/src'
make[2]: *** [Makefile:318: all] Error 2
make[2]: Leaving directory '/home/pi/gmrender-resurrect/src'
make[1]: *** [Makefile:370: all-recursive] Error 1
make[1]: Leaving directory '/home/pi/gmrender-resurrect'
make: *** [Makefile:311: all] Error 2
```

--- a/src/upnp_compat.h
+++ b/src/upnp_compat.h
@@ -29,12 +29,12 @@
 
 #if UPNP_VERSION >= 11000
 #define UpnpAddVirtualDir(x) UpnpAddVirtualDir(x, NULL, NULL)
-#define VD_GET_INFO_CALLBACK(NAME, FILENAME, INFO, COOKIE) int NAME(const char* FILENAME, UpnpFileInfo* INFO, const void* COOKIE, const void **)
-#define VD_OPEN_CALLBACK(NAME, FILENAME, MODE, COOKIE) UpnpWebFileHandle NAME(const char* FILENAME, enum UpnpOpenFileMode MODE, const void* COOKIE, const void *)
-#define VD_READ_CALLBACK(NAME, HANDLE, BUFFER, LENGTH, COOKIE) int NAME(UpnpWebFileHandle HANDLE, char* BUFFER, size_t LENGTH, const void* COOKIE, const void *)
+#define VD_GET_INFO_CALLBACK(NAME, FILENAME, INFO, COOKIE) int NAME(const char* FILENAME, UpnpFileInfo* INFO, const void* COOKIE, const void ** REQUEST_COOKIE)
+#define VD_OPEN_CALLBACK(NAME, FILENAME, MODE, COOKIE) UpnpWebFileHandle NAME(const char* FILENAME, enum UpnpOpenFileMode MODE, const void* COOKIE, const void * REQUEST_COOKIE)
+#define VD_READ_CALLBACK(NAME, HANDLE, BUFFER, LENGTH, COOKIE) int NAME(UpnpWebFileHandle HANDLE, char* BUFFER, size_t LENGTH, const void* COOKIE, const void * REQUEST_COOKIE)
 #define VD_WRITE_CALLBACK(...) VD_READ_CALLBACK(__VA_ARGS__)
-#define VD_SEEK_CALLBACK(NAME, HANDLE, OFFSET, ORIGIN, COOKIE) int NAME(UpnpWebFileHandle HANDLE, off_t OFFSET, int ORIGIN, const void* COOKIE, const void *)
-#define VD_CLOSE_CALLBACK(NAME, HANDLE, COOKIE) int NAME(UpnpWebFileHandle HANDLE, const void* COOKIE, const void *)
+#define VD_SEEK_CALLBACK(NAME, HANDLE, OFFSET, ORIGIN, COOKIE) int NAME(UpnpWebFileHandle HANDLE, off_t OFFSET, int ORIGIN, const void* COOKIE, const void * REQUEST_COOKIE)
+#define VD_CLOSE_CALLBACK(NAME, HANDLE, COOKIE) int NAME(UpnpWebFileHandle HANDLE, const void* COOKIE, const void * REQUEST_COOKIE)
 #elif UPNP_VERSION >= 10803
 #define UpnpAddVirtualDir(x) UpnpAddVirtualDir(x, NULL, NULL)
 #define VD_GET_INFO_CALLBACK(NAME, FILENAME, INFO, COOKIE) int NAME(const char* FILENAME, UpnpFileInfo* INFO, const void* COOKIE)

